app/forms/journeys/additional_payments_for_teaching/eligible_itt_subject_form.rb

```diff
@@ -8,15 +8,15 @@ class EligibleIttSubjectForm < Form
       validates :eligible_itt_subject, inclusion: {in: :available_options, message: i18n_error_message(:inclusion)}
 
       def available_subjects
-        @available_subjects ||= subject_symbols(claim).map(&:to_s)
+        @available_subjects ||= subject_symbols(current_claim: claim, answers: journey_session.answers).map(&:to_s)
```

Comment:
I think we need to refactor `subject_symbols` so we aren't passing a claim object to it. We should only need to pass in the specific answers needed to calculate the available subjects.